shardingsphere-features/shardingsphere-db-discovery/shardingsphere-db-discovery-provider/pom.xml

```diff
@@ -31,5 +31,6 @@
     
     <modules>
         <module>shardingsphere-db-discovery-mgr</module>
+        <module>shardingsphere-db-discovery-og</module>
```

Comment:
I think `og` is not a good name. ^_^

Reply:
shardingsphere-db-discovery-openGauss？ any suggest?

Reply:
Now, the exist module' name is `opengauss`, for example `shardingsphere-sql-parser-opengauss`,  `shardingsphere-db-protocol-opengauss`.

I suggest we can keep the same name.